dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,19 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recover from failure in Expanding volumes
+
+If for some reason expanding underlying storage fails either on master or on node,
+you may want to manually recover PVC state and cancel resize requests that will be
+continuously retried by controller without admin intervention.
+
+Currently this can be done manually via:
+
+1. Mark PersistentVolume(PV) that is bound to the claim(PVC) with "Retain" reclaim policy. This can be
+   done by editing the PV and changing `persistentVolumeReclaimPolicy` to `Retain`.
+2. Delete the PVC.
+3. Re-create the PVC with smaller size or size that can be allocated by the underlying storage provider.
```

Comment:
"Re-create the PVC in a smaller size or a size that can be allocated by the underlying storage provider."